src/routes/index.js

```diff
@@ -21,4 +22,6 @@ router.use('/profiles', userRouter);
 router.use('/profiles', followRouter);
 router.use('/users', userRouter);
 
+router.use('/roles', rolesRouter);
```

Comment:
This route should be tied to the users route; `/users/roles`.

Reply:
feedback implemented